api/src/chat/services/bot.service.ts

```diff
@@ -252,13 +252,13 @@ export class BotService {
           assign_labels: [],
           trigger_labels: [],
           attachedBlock: undefined,
-          category: undefined,
+          category: undefined as any,
```

Comment:
```suggestion
          category: null,
```

Reply:
The current typing don't support the null value 
![image](https://github.com/user-attachments/assets/5d855a7d-dba2-4816-a968-47a4c0ad7fb6)

Reply:
This update is applied :heavy_check_mark: